packages/theme-check-node/src/index.ts

```diff
@@ -127,6 +146,7 @@ export async function themeCheckRun(
     getSectionSchema: async (name) => sectionSchemas.get(name)?.(),
     getBlockSchema: async (name) => blockSchemas.get(name)?.(),
     getAppBlockSchema: async (name) => blockSchemas.get(name)?.() as any, // cheating... but TODO
+    getDocDefinition: async (uri) => docDefinitions.get(uri)?.(),
```

Comment:
Last nit is probably that this should be either a relative path or a name instead?